common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -113,7 +113,8 @@ public static String getVersion() {
     }
 
     public static boolean isPreAlpha() {
-        return version.contains("pre-alpha");
+        // This is hard-coded in the development branch
+        return true;
```

Comment:
We will have to not forget to revert this

Reply:
Hopefully we will get a merge conflict here. If you want to be sure that we do, I can make a separate change to main, hardcoding it to `false`.

Reply:
If you want to go ahead, but I won't require it.